Stylesheets/Components/avatar.css

```diff
@@ -0,0 +1,47 @@
+.avatar-circle {
+    max-width: 100%;
+    height: auto;
+    border-radius: 50%;
+    height: 2.5rem;
+    width: 2.5rem;
+}
+
+.avatar-square {
+    max-width: 100%;
+    height: auto;
```

Comment:
remove height: auto as it is overridden by the below height: 2.5rem

Reply:
Thanks for pointing it out.
have made the suitable changes